specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -337,6 +337,22 @@ not required to upload.
 
   @doc("Configuration for quick load test")
   optionalLoadTestConfig?: OptionalLoadTestConfig;
+
+  @added(APIVersions.v2023_04_01_preview)
+  @doc("Configure engine instances to be split into multiple regions")
+  regionConfiguration?: RegionConfig[];
+}
+
+@added(APIVersions.v2023_04_01_preview)
+@doc("Region configuration for load test")
+model RegionConfig {
+  @doc("Region to apply this configuration")
+  regionName?: string;
+
+  @doc("""
+  The number of engine instances to execute load test in this region. Supported values are in range of 1-400.
+  """)
+  engineInstances?: int32;
 }
```

Comment:
Both are optional? What if someone passes `regionConfiguration: [{}]`? Seems the `regionName` should at least be required, if not both properties. I don't understand the use case of allowing both to be elided.